UPSTREAM_PROJECTS.yaml

```diff
@@ -195,7 +195,7 @@ projects:
           - commit: 7b286fdc8e8fa91a6e9a179a5494b6ee29fce17b
       - name: hook
         versions:
-          - commit: 6d43b8b331c7a389f3ffeaa388fa9aa98248d7a2
+          - tag: ad7836d56d9a3152439daa3085b233ee4e13d886
```

Comment:
is this an actual tag? It looks like a commit hash?

Reply:
it is a commit, but the CI failed when it was a commit, so i switched it. I'm trying to find the prow job to show the failure.

Reply:
oh yeah I see it [here](https://prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp.s3.us-west-2.amazonaws.com/pr-logs/pull/aws_eks-anywhere-build-tooling/1762/validate-generated-presubmit/1616557262220824576/build-container-build-log.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAUTLHWUYSHC5LKUOS%2F20230123%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20230123T222912Z&X-Amz-Expires=600&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEHYaCXVzLXdlc3QtMiJIMEYCIQDfLJrhT7lg7JMUlK%2Fn8%2Bz1IW%2B4Fia42cQ9vRiOcWAHIAIhAIxfyw3yDPuoI0S1o9MkArpPuyiwz%2BysC8a34OokHwskKvkECN%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEQARoMMzE2NDM0NDU4MTQ4Igy%2FmUv14hM%2BBGXB4h8qzQR0R1cclgMU7QJMXk3W6i5YrYLtrTJgucBN7yHwl3fg77Fj7dBYPf7%2BxsJpglCk1Nd9dfX6UQjuB1aEEDEjoZ86NPkFb4fn0WBCVcioQ%2FsJcwIdjkr3NLSM2ucwRjXpi%2B0ZO5IyRVVRYHswtt5DY1bMw%2FISywML80l0SwLY%2BjyhnUR1oW9oj0OupDvK%2BxAjqTqrgD5rBc9Xgzo1UTL8bUa%2BqGnWSVa6uExS%2BoEDWOvgU1pNyfaaAztoJR2OUQfzR2GtPaKVa9IzPpIdPGEA26x76DyUa34xYGHuf5IY4BC9uQ3FvcaVfLppFNpczY3ZFDbXRuVlgaqKdcxLxNazVAT0eftVxEOTTMha8%2BExMnoQ7NPBU7dgMKg4xCHzQBuIVeB4eUa%2F8HpWG32aFKPEL6w9kHEEwJ9xVDN0Jv438oHFG37sT7tSP7c1tSl7lxNfHc0%2FhHkuSqM4sRn4B4DNdZ7UG8G5xI9iIbIfzaZhTBIZXEtadMleFazs0bp6fBsBI05%2FP0mFru%2FvKY3YfAijJrW4n3IFjIhvwKV2A%2BDfe4mbJwj5WyjldZzgXczMMpu97bo9rL9EEE1ntB9ISmU4WGbIgZx8PjlMC5T66nBcS9qjdIT77njrVD9EUBx%2BMZnUbgpt%2FdIUwHSfzCSnLRtQapnyL1uBxz1vbHTPcjL1GTmXwcxtKkyPJ0H1nQ3ltOuvcWpramdYzrRRnszemO1wnVTzap137Ey9FxXo%2BlR24xSxqUA8oDPkTQEVcLbG7IuzXyqH9CBxmsyGsUew%2BfI1MKCMvJ4GOpkBIff8Tbqla8eMlEgMPSN9Cx6xvupQyoe%2FlGa7H3Ofe79BpIYP2H7%2FdOGvn06MHBJxMxdWzGITtPK9ueWpLXslTR0Dor%2Fi0qoj15bqL7%2FGpZ9ixHJZlgiy%2B3kln340DdeMURupi657UUfqw%2B03j%2FIBkQWCQYXV%2BRiyPmN0gQ0WOS1KrsvDFPdLDcH5XbUmzEJx6SmvGYiYJGYR&X-Amz-SignedHeaders=host&X-Amz-Signature=2027fde887f16719dacac3b13671c740145575ac4862119b6a306592e8b4e73f) 
It's strange why it expects tag there instead of commit